packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -83,17 +103,25 @@ @interface FLTCam : NSObject <FlutterTexture,
 @property(strong, nonatomic) AVCaptureAudioDataOutput *audioOutput;
 @property(assign, nonatomic) BOOL isRecording;
 @property(assign, nonatomic) BOOL isAudioSetup;
+@property(assign, nonatomic) BOOL isStreamingBytes;
 - (instancetype)initWithCameraName:(NSString *)cameraName
                   resolutionPreset:(NSString *)resolutionPreset
                              error:(NSError **)error;
+
 - (void)start;
 - (void)stop;
 - (void)startVideoRecordingAtPath:(NSString *)path result:(FlutterResult)result;
 - (void)stopVideoRecordingWithResult:(FlutterResult)result;
+- (void)startByteStreamWithMessenger:(NSObject<FlutterBinaryMessenger> *)messenger;
+- (void)stopByteStream;
 - (void)captureToFile:(NSString *)filename result:(FlutterResult)result;
 @end
 
 @implementation FLTCam
+FourCharCode const videoFormat = kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange;
```

Comment:
Format that I believe is recommended for iOS 10+, which is minimum requirement for this plugin.